rectools/models/lightfm.py

```diff
@@ -130,7 +130,7 @@ def __init__(
         self.model: LightFM
         self._model = model
         self.n_epochs = epochs
-        self.n_threads = num_threads
+        self.n_threads = num_threads  # type: ignore  # TODO: remove when recommend n_threads implemented
```

Comment:
That's tricky actually. We are using it in LightFM for fit also. Let's please remove this comment here and discuss it in our `recommend n_threads` PR.
Also it is more menaingfull to move this comment to vector.py